en/docs/setup/data-purging.md

```diff
@@ -238,6 +238,13 @@ You can perform data purging by clearing the session data using the script given
     !!! info
 
         For DB2, MySQL, Oracle, and Postgre database scripts, see [Stored Procedures](https://github.com/wso2/carbon-identity-framework/tree/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures).
+   
+        You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the time created or the expired time. By default, the session clean up task cleans the data based on the time session was created.
```

Comment:
How this renders properly in the build.

Reply:
Is this only applicable to 5.10?

Reply:
This is applicable to other versions too. This is the reported version. Once this is reviewed and approved, will do the same changes for other versions too.

Reply:
It renders properly. But there is an issue in the sidebar rendering in the local setup. That should be a issue in the local dev setup
<img width="1344" alt="Screen Shot 2022-03-04 at 10 22 06 AM" src="https://user-images.githubusercontent.com/17597293/156701694-14a1dd29-ad81-4df5-bda7-3ba7e6808b62.png">